crypto/rand_extra/ccrandomgeneratebytes_test.cc

```diff
@@ -0,0 +1,39 @@
+#if !defined(_DEFAULT_SOURCE)
+#define _DEFAULT_SOURCE  // Needed for getentropy on musl and glibc
+#endif
+
+#include <openssl/rand.h>
+
+#include "internal.h"
+
+#if defined(OPENSSL_RAND_CCRANDOMGENERATEBYTES)
+
+#include <gtest/gtest.h>
+
+#include <openssl/span.h>
+
+#include <CommonCrypto/CommonRandom.h>
+
+#include "../test/test_util.h"
+
+// This test is, strictly speaking, flaky, but we use large enough buffers
+// that the probability of failing when we should pass is negligible.
+
+TEST(CCRandomGenerateBytesTest, NotObviouslyBroken) {
+  static const uint8_t kZeros[1024] = {0};
+
+  uint8_t buf1[256], buf2[1024], buf3[256];;
```

Comment:
NP: to make the check on line 30/31 more interesting you could zero initilize these arrays, the check you do on 34-36 covers that. You also have an extra semicolon.